tests/spec/core/utils-spec.js

```diff
@@ -239,32 +239,6 @@ describe("Core - Utils", () => {
     });
   });
 
-  describe("humanDate", () => {
```

Comment:
Should move W3CDate tests here now that it's part of core/utils?

Reply:
Probably, yeah.

Reply:
We don't have specific tests for W3CDate, so let's ignore this.